friendshipper/src/lib/components/oobe/WelcomeModal.svelte

```diff
@@ -74,15 +74,22 @@
 					awsSsoStartUrl !== '' &&
 					awsRoleName !== '' &&
 					artifactBucket !== '';
+				awsAccountId = awsAccountId.replaceAll(' ', '');
```

Comment:
I think string in javascript has trim() which gets rid of all leading and trailing whitespace. wdyt about using that?